src/window.rs

```diff
@@ -865,6 +865,7 @@ pub enum Fullscreen {
 
 #[derive(Clone, Debug, PartialEq)]
 pub enum Theme {
+    System,
```

Comment:
I don't like that change, and still prefer the use of `Option`, since we use `Theme` in `ThemeChanged` event, and so if the user  gets `System` it is confusing, since they don't know how to adapt to it. And even if winit can't send `System` users are still have to do something about it, so let's go back to `Option` approach to avoid that confusion.

Reply:
Note, you can internally map `None` to `System`, but you shouldn't expose that API to the user, since users don't have a way to handle `System` variant.